app/build.gradle

```diff
@@ -211,6 +211,8 @@ dependencies {
         transitive = true
     }
 
+    //UI Component : Expandable Layout
+    implementation "net.cachapa.expandablelayout:expandablelayout:$rootProject.expandableLayoutVersion"
```

Comment:
Would it be possible to try implement this natively rather than using a library? I think we should try use as little third party libraries as possible.

Reply:
@jaztriumph sorry that I only seen this now and for the delay in getting back to you. I think the feature looks good so if you would like to go ahead with implementation, I would be happy with it